### PR TITLE
[Trivial] BIP9 parameters on regtest cleanup

### PR DESCRIPTION
**Pull request by paveljanik**
Minor cleanup after https://github.com/bitcoin/bitcoin/commit/56c87e92110f05d7452f1e85bf755246ffc77206

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -996,10 +996,10 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                 return InitError(strprintf("Invalid nTimeout (%s)", vDeploymentParams[2]));
             }
             bool found = false;
-            for (int i=0; i<(int)Consensus::MAX_VERSION_BITS_DEPLOYMENTS; ++i)
+            for (int j=0; j<(int)Consensus::MAX_VERSION_BITS_DEPLOYMENTS; ++j)
```
Comment (btcdrak) [inline src/init.cpp]: Why?

Comment (paveljanik) [inline src/init.cpp]: @btcdrak: See #8105. There is `auto i` a few lines above: https://github.com/bitcoin/bitcoin/blob/master/src/init.cpp#L985

**Comment by NicolasDorier** [comment]
utACK 0fc00bea5d24f4a6715f7069adb7b7396c981308

**Comment by MarcoFalke** [comment]
utACK 0fc00be

**Comment by sipa** [comment]
utACK 0fc00bea5d24f4a6715f7069adb7b7396c981308

**Comment by laanwj** [comment]
utACK 0fc00be

**Pull request merged**